docs/_posts/2019-05-08-530-release.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blogpost
+title:  "5.3.0 Release Announcement"
+section_id: blog
+date:   2019-05-08 12:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hello Open GEE Community!
+ 
+We are happy to announce the official release of Open GEE 5.3.0!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+ 
+**Performance and memory enhancements include:**
+
+**- The ability to control the XML cache size.** The size can be modified by configuring the _INIT_HEAP_SIZE_, _MAX_HEAP_SIZE_, and _BLOCK_SIZE_ settings in _/etc/opt/google/XMLparams_. This file must be created manually. View the [Configure Fusion performance document](http://www.opengee.org/geedocs/5.3.0/answer/176738.html) for more details.
+
+**- The capability of purging periodically the XML cache.** This can be done by configuring the PURGE and PURGE_LEVEL settings in /etc/opt/google/XMLparams. This file must be created manually. The details are explained in the [Configure Fusion performance document](http://www.opengee.org/geedocs/5.3.0/answer/176738.html).
+
+**- Improvement of system manager performance for large builds.** This release eliminates some duplicate processing in system manager, which reduces the amount of time it takes to initiate large builds.
+
+**Build and install enhancements:**
+
+**- Option to pass long_version parameter to the SCons OpenGEE build.** This enables one to override the long version string.
+**- Options to the Server install script.** The server install script now checks the size of the publish root and allows the user to set custom user and group names.
+
+**Third party libraries upgrades:**
+
+**-Upgraded OpenJPEG 2000 library to latest version: 2.3.1.** This upgrade brings multiple bugs fixes and improvements.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.3.0-RC3). The full release notes can also be found [here](https://www.opengee.org/geedocs/answer/7160007.html).
```

Comment:
The ulr of final build is not correct. It will be fixed once we have the final 5.3.0 build.